client/post-editor/editor-revisions-list/style.scss

```diff
@@ -36,14 +36,11 @@
 }
 
 .editor-revisions-list__header {
-	$editor-revisions-list-header-height: 46px;
-	$editor-revisions-list-header-font-size: 16px;
 	padding: 0 16px;
-	height: $editor-revisions-list-header-height;
-	line-height: $editor-revisions-list-header-height;
+	height: 46px;
```

Comment:
Some of the pixel values seem a bit arbitrary -- can we calculate them based on other variables? Does this change merit setting any variable(s)?

Reply:
As I saw it, the variables in the original were useful in vertically centring the revisions count and giving vertical position and height to the `.is-loading` element. This vertical centring didn't look right when these elements were sharing a visual block with the nav buttons: it seemed to make more sense to bring them closer together, so I used literal values to achieve the right vertical position. Following Shaun's new mockup, we can vertically centre each part in its own container, which will allow me to use variables again.